src/OrchardCore.Modules/OrchardCore.Contents/Views/Contents-Button-Actions.SummaryAdmin.cshtml

```diff
@@ -6,32 +6,17 @@
     var hasPublished = await ContentManager.HasPublishedVersionAsync(contentItem);
     var hasDraft = contentItem.HasDraft();
     var hasClonePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.CloneContent, contentItem);
+    var hasDeletePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.DeleteContent, contentItem);
+    var hasEditPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.EditContent, contentItem);
+    var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PreviewContent, contentItem);
+    var hasPublishPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PublishContent, contentItem);
 }
 
-@if (hasDraft)
-{
-    <a asp-action="Publish" asp-controller="Admin" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" asp-route-returnUrl="@FullRequestPath" itemprop="UnsafeUrl" class="dropdown-item btn-sm">@T["Publish Draft"]</a>
-    <a asp-action="Preview" asp-controller="Item" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" class="dropdown-item btn-sm">@T["Preview Draft"]</a>
+<a asp-action="Publish" asp-controller="Admin" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" asp-route-returnUrl="@FullRequestPath" itemprop="UnsafeUrl" class="dropdown-item btn-sm @(hasDraft && hasPublishPermission)? string.Empty : "disabled")">@T["Publish Draft"]</a>
+<a asp-action="Preview" asp-controller="Item" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" class="dropdown-item btn-sm @(hasDraft && hasPreviewPermission? string.Empty : "disabled")">@T["Preview Draft"]</a>
+<div class="dropdown-divider"></div>
 
-    <div class="dropdown-divider"></div>
-}
-
-@if (hasDraft && hasPublished)
```

Comment:
Please don't change this behavior, I assume it was decided to not show the actions if they don't apply

Reply:
Ok

Reply:
> I assume it was decided to not show the actions if they don't apply

Dean prefers to hide the actions instead of disable them too, but when I hide the other actions before the commit the UI becomes ugly :( I can provide a screenshot

Reply:
![UI Pemissions](https://user-images.githubusercontent.com/3237266/90197509-8c504780-ddd7-11ea-8894-3540416bb785.png)

Reply:
I don't see the problem with hiding the `Edit` button here if the user does not have permission.

It is consistent to hide with all the other permission related ui, i.e. the menu looks pretty bare to, but that is what you would expect when you don't have permissions to execute an operation.

If you try to change this behaviour it will have to many ripple effects in other areas @hishamco

Reply:
I don't have any problems at all, but I find the UI/UX is not consistent, so I will hide the actions then